docs/user_guide.md

```diff
@@ -39,108 +41,145 @@ Sample usage:
 ```
 $ gradlew run -Dargs="-config CSV_path.csv -output output_path/ -since 01/10/2017 -until 01/11/2017"
 ```
+
 `-Dargs="..."` uses the same argument format as mentioned above.
 
 ```
 Note: 
-The contribution calculation is base on the daily commits made within 00:00 to 23:59 in GMT+8.
+The contribution calculation is based on the daily commits made within 00:00 to 23:59 in GMT+8.
 ```
+
+
 ## CSV Config File
-The CSV Config files control the list of target repositories. It also contains a white list of authors(if the author is listed in the CSV, his/her contribution will be ignored by the analyzer.)
-[Sample_full.csv](../sample_full.csv) in root is an example CSV config file. It should contain the following columns:
+The `CSV Config File` contains the list of repositories, and the corresponding target authors to track contribution of.
+[Sample_full.csv](../sample_full.csv) is an example of a configuration file setup. It should contain the following columns:
 
 Column Name | Explanation
 ----------- | -----------
 Organization | Organization of the target repository
 Repository | Name of the target repository
-branch | Target branch
-StudentX's Github ID | Author's Github ID.
-StudentX's Display Name | Optional Field. The value of this field, if not empty,will be displayed in the dashboard instead of author's Github ID.
-StudentX's Local Author Name | Detailed explanation below
-
-## Preparation of Repositories
-### Local Git Author Name
-First, what is Git Author Name?
-
-Git Author Name refers to the customizable Git Author Display Name set in local .gitconfig file. It will be displayed as Author name and Committer Name in Git. For example, in Git Log output:
+Branch | The branch to analyse in the target repository
+Author's GitHub ID | GitHub ID of the target contributor in the repository
+Author's Display Name | Optional Field. The value of this field, if not empty, will be displayed in the dashboard instead of author's GitHub ID.
+[Optional] Author's Git Author Name | Detailed explanation below
+
+#### Git Author Name
+`Git Author Name` refers to the customizable author's display name set in the local `.gitconfig` file.
+It is displayed as author name as opposed to the `GitHub ID` in the entries.
+For example, in the Git Log's display:
 ```
 ...
 commit cd7f610e0becbdf331d5231887d8010a689f87c7
-Author: fakeAuthor <ma.tanghao@dhs.sg>
+Author: ConfiguredAuthorName <author@example.com>
 Date:   Fri Feb 9 19:14:41 2018 +0800
 
-    moved
+    Make some changes to show my new author's name
 
 commit e3f699fd4ef128eebce98d5b4e5b3bb06a512f49
-Author: harryggg <ma.tanghao@dhs.sg>
+Author: ActualGitHubId <author@example.com>
 Date:   Fri Feb 9 19:13:13 2018 +0800
 
-    new
+    Initial commit
  ...
 ```
-*fakeAuthor* and *harryggg* are both Local Git Author Name.
+**ActualGitHubId** and **ConfiguredAuthorName** are both `Git Author Name`.
```

Comment:
For clarity, add ```of the same author.```

**ActualGitHubId** and **ConfiguredAuthorName** are both `Git Author Name` of the same author.